src/cmd/flux-jobs.py

```diff
@@ -331,6 +317,13 @@ class JobsOutputFormat(flux.util.OutputFormat):
     a new format suitable for headers display, etc...
     """
 
+    class JobFormatter(string.Formatter):
+        def format_field(self, value, spec):
+            if spec.endswith("h"):
+                value = "-" if value in ("", "0s") else str(value)
```

Comment:
I gave `{t_inactive:>12.8h} {runtime_hms:h}` a try and I wonder if we should add `0.0` and `0:00:00` to this check:

```
❯ flux jobs -a -o "{t_inactive:>12.8h} {runtime_hms:h}"                                                                               11:08:51 ()
    T_INACTI RUNTIME
         0.0 0:00:00
         0.0 0:00:00
         0.0 0:00:00
         0.0 0:56:20
         0.0 0:56:43
    15849862 0:00:01

```

Reply:
Hmmm, that's a good point, now that we can just "throw" `:h` on any field, there are more output options. Let me see if that works out.